### ISSUE TITLE
False positive for [MD051] “Link fragments should be valid”

### ISSUE THREAD

**Issue by jan-ferdinand** [joshrotenberg/mdbook-lint#324]
Consider the following section in an mdbook:

```md
## `a` + `title`

[somewhere](#a--title)
```

Running mdbook-lint incorrectly triggers [MD051]. At least I _think_ it’s a false positive, because the slug `a--title` is what mdbook produces, and it works as an anchor without problem. Perhaps this is a bug in the slugification of mdbook? In any case, I think mdbook-lint should be consistent with mdbook.
